functests/sctp/sctp.go

```diff
@@ -43,12 +49,15 @@ type nodesInfo struct {
 
 func init() {
 	roleWorkerCNF := os.Getenv("ROLE_WORKER_CNF")
-	if roleWorkerCNF == "" {
-		roleWorkerCNF = "worker-cnf"
+	if roleWorkerCNF != "" {
+		sctpNodeSelector = fmt.Sprintf("node-role.kubernetes.io/%s=", roleWorkerCNF)
```

Comment:
Bear in mind that here you are in `init()` which is the module initialization.
I would not rely on the fact that this part gets triggered before the client one. If the order is not correct, you have the k8s client not inited yet.
Moreover, you are doing the same in `beforeAll`